package.json

```diff
@@ -12,7 +12,7 @@
     "type-check": "nx affected --target=type-check",
     "preinstall": "node ./tools/scripts/preinstall.mjs",
     "postinstall": "patch-package",
-    "dedupe": "npx yarn-deduplicate --strategy fewer"
+    "dedupe": "npx yarn-deduplicate --strategy highest"
```

Comment:
will dedupe to highest possible semver range -> thus all packages in our repo are bumped as part of this PR